translations/translations.json

```diff
@@ -3059,6 +3059,7 @@
       "thanksMessage": "Dein Feedback wurde erfolgreich abgesendet.",
       "negativeRating": "Diese Seite ist nicht hilfreich",
       "positiveRating": "Diese Seite ist hilfreich",
+      "searchTermDescription": "Für den folgenden Begriff konnte kein Ergebnis gefunden werden:",
```

Comment:
Not sure if we shouldn't rather change this to something like just `Suchbegriff` or `Fehlender Inhalt`.

Reply:
I'm fine with the text as it is, actually.